src/basic_shapes.rs

```diff
@@ -0,0 +1,63 @@
+//! An example of generating basic shapes
+extern crate image;
+extern crate num_complex;
+extern crate line_drawing;
+
+fn circle_bres(imgbuf: image::RgbaImage, xc: i32, yc: i32, r: i32) -> image::RgbaImage {
```

Comment:
Change this to take `imgbuf` as `&mut`: `circle_bres(imgbuf: &mut image::RgbaImage, xc: i32, yc: i32, r: i32)`